jiant/tasks/lib/qasrl.py

```diff
@@ -0,0 +1,81 @@
+import gzip
+import nltk
+import json
+
+from jiant.tasks.lib.templates import span_prediction as span_pred_template
+from jiant.utils.retokenize import TokenAligner
+
+
+class QASRLTask(span_pred_template.AbstractSpanPredicationTask):
+    def get_train_examples(self):
+        return self._create_examples(self.train_path, set_type="train")
+
+    def get_val_examples(self):
+        return self._create_examples(self.val_path, set_type="val")
+
+    def get_test_examples(self):
+        return self._create_examples(self.test_path, set_type="test")
+
+    def _create_examples(self, file_path, set_type):
+
+        with gzip.open(file_path) as f:
+            lines = f.read().splitlines()
+
+        examples = []
+        ptb_detokenizer = nltk.tokenize.treebank.TreebankWordDetokenizer()
+
+        for line in lines:
+            datum = json.loads(line)
+            datum = {
+                "sentence_tokens": datum["sentenceTokens"],
+                "entries": [
+                    {
+                        "verb": verb_entry["verbInflectedForms"]["stem"],
+                        "verb_idx": verb_idx,
+                        "questions": {
+                            question: [
+                                [
+                                    {
+                                        "tokens": datum["sentenceTokens"][span[0] : span[1] + 1],
```

Comment:
Based on the source data, it looks like `span[1] + 1` will overshoot the span. Can you check on this?

Reply:
It actually does. Can't believe it's wrong all this time, one more thing to wipe out at the north pole.